src/format/reveal/format-reveal.ts

```diff
@@ -288,8 +289,7 @@ function revealHtmlPostprocessor(format: Format) {
 
     // remove all attributes from slide headings (pandoc has already moved
     // them to the enclosing section)
-    const slideLevel = parseInt(format.metadata[kSlideLevel] as string, 10) ||
-      2;
+    const slideLevel = format.pandoc[kSlideLevel] || 2;
```

Comment:
This has nothing to do with the feature it is only fixing a previous retrieval that was not correct as `slide-level` should be in Pandoc